DoctrineFixturesBundle.php

```diff
@@ -24,4 +25,11 @@
  */
 class DoctrineFixturesBundle extends Bundle
 {
+    /**
+     * {@inheritDoc}
+     */
+    public function registerCommands(Application $application)
+    {
+        // noop
```

Comment:
As I said in the related pull request made by @chalasr in Symfony repo, this may be technically correct but it looks super strange. A method to register commands that must be completely empty to register the commands?

Reply:
You're right of course. Removed.